bigquery/google/cloud/bigquery/job.py

```diff
@@ -506,6 +529,11 @@ def output_rows(self):
     https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load.allowQuotedNewlines
     """
 
+    autodetect_schema = AutoDetectSchema('autodetect_schema', bool)
```

Comment:
Since it's just called `autodetect` in the backend API, I'd prefer that the client library stay consistent with that.